src/msw/uilocale.cpp

```diff
@@ -65,18 +74,49 @@ void wxUseLCID(LCID lcid)
 }
 
 // ----------------------------------------------------------------------------
-// wxUILocale implementation for MSW
+// wxLocaleIdent::GetName() implementation for MSW
 // ----------------------------------------------------------------------------
 
-// TODO-XP: Replace this with an implementation using GetLocaleInfoEx() when we
-// don't support XP any longer.
+wxString wxLocaleIdent::GetName() const
+{
+    // Construct name in right format:
+    // Windows: <language>-<script>-<REGION>
+
+    wxString name;
+    if (!m_language.empty())
+    {
+        name << m_language;
+
+        if (!m_script.empty())
```

Comment:
```suggestion
        if ( !m_script.empty() )
```

Reply:
I had originally decided it wasn't worth fixing the style in the original submission, but let's do it if I'm not the only bothered by it.